.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Feature request
 about: Suggest an idea for this project
-labels: feature, request, under discussion, priority: undecided
+labels: 'feature, request, under discussion,'priority: undecided'
```

Comment:
1. There are 3 `'`
2. Are you sure the `'` should not be around each of the labels?

Reply:
I take back the 2.